doc/src/compute_rattlers.rst

```diff
@@ -0,0 +1,77 @@
+.. index:: compute rattlers
+
+compute rattlers command
+========================
+
+Syntax
+""""""
+
+.. parsed-literal::
+
+   compute ID group-ID rattlers cutoff zmin ntries
+
+* ID, group-ID are documented in :doc:`compute <compute>` command
+* rattlers = style name of this compute command
+* cutoff = *type* or *radius*
+
+  .. parsed-literal::
+
+       *type* = cutoffs determined based on atom types
+       *radius* = cutoffs determined based on atom diameters (atom style sphere)
+
+* zmin = minimum coordination for a non-rattler particle
+* ntries = maximum number of iterations to remove rattlers
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   compute 1 all rattlers type 4 10
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Define a compute that identifies rattlers in a system. Rattlers are
+identified using an interactive approach. The coordination number of
+all atoms is first calculated.  The *type* and *radius* settings are
+used to select whether interaction cutoffs are determined by atom
+types or by the sum of atomic radii (atom style sphere), respectively.
+Rattlers are then identified as particles with a coordination number
+less than *zmin* and are removed from consideration. Atomic coordination
+numbers are then recalculated, excluding previously identified rattlers,
+to identify a new set of rattlers. This process is iterated, up to a maximum
+of *ntries*, until no new rattlers are identified and the remaining
```

Comment:
I think a sentence or two of guidance about setting ntries would be useful.  Possibly it is different for 2d vs 3d systems?  Or for crystalline solids vs glasses, etc ?

Reply:
I added some guidance but cautioned readers that it really depends on the system. E.g. a linear chain of N particles would take N/2 iterations to sequentially pop particles off the ends until it discovers that the entire chain is unstable (for a minimum coordination of 2).